test/integration/cve_int_test.go

```diff
@@ -26,16 +26,38 @@ func (suite *CveIntegrationTestSuite) TestCveSummary() {
 	cp.Expect("ActivePython-3.7")
 	cp.Expect("0b87e7a4-dc62-46fd-825b-9c35a53fe0a2")
 
-	cp.Expect("37 Vulnerabilities")
+	cp.Expect("Vulnerabilities")
 	cp.Expect("6")
 	cp.Expect("CRITICAL")
-	cp.Expect("10 Affected Packages")
+	cp.Expect("13 Affected Packages")
 	cp.Expect("tensorflow")
 	cp.Expect("1.12.0")
 	cp.Expect("18")
 	cp.ExpectExitCode(0)
 }
 
+func (suite *CveIntegrationTestSuite) TestCveReport() {
+	suite.OnlyRunForTags(tagsuite.Cve)
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	ts.LoginAsPersistentUser()
+
+	cp := ts.Spawn("cve", "report", "ActiveState/ActivePython-3.7")
```

Comment:
This project isn't owned by us, and could be updated to patch these vulnerabilities. You may want to fork this project, or create your own so this test does not end up randomly breaking